# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rox-filer/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Yet another hack for this hacky installation ...

  - Rene Rebe <rene@exactcode.de>

--- rox-2.1.3/install.sh	2003-11-23 17:16:58.000000000 +0100
+++ rox-2.1.3/install.sh	2004-09-29 16:46:41.265314758 +0200
@@ -203,9 +203,9 @@
 endir "$MIMEDIR"
 endir "$MIMEDIR/packages"
 cp rox.xml "$MIMEINFO" || die "Failed to create $MIMEINFO"
-update-mime-database "$MIMEDIR" || die "update-mime-database failed
-Make sure you have installed the shared MIME database from:
-http://www.freedesktop.org/software/shared-mime-info"
+#update-mime-database "$MIMEDIR" || die "update-mime-database failed
+#Make sure you have installed the shared MIME database from:
+#http://www.freedesktop.org/software/shared-mime-info"
 
 echo "Installing application..."
 endir "$APPDIR"
